docs/content/dagster-cloud/deployment/agents/amazon-ecs/configuration-reference.mdx

```diff
@@ -0,0 +1,190 @@
+
+platform_type: "cloud"
+---
+
+# Amazon ECS agent configuration reference
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+This reference describes the various configuration options Dagster Cloud currently supports for [Amazon ECS agents](/dagster-cloud/deployment/agents/amazon-ecs).
+
+---
+
+## Per-location configuration: Secrets and environment variables
+
+When [adding a code location](/dagster-cloud/developing-testing/code-locations) to Dagster Cloud with an Amazon ECS agent, you can use the `container_context` key on the location configuration to add additional ECS-specific configuration that will be applied to any ECS tasks associated with that code location. The following example illustrates the available fields:
+
+```yaml
+location_name: cloud-examples
+image: dagster/dagster-cloud-examples:latest
+code_source:
+  package_name: dagster_cloud_examples
+container_context:
+  ecs:
+    env_vars:
+      - FOO_ENV_VAR=foo_value
+      - BAR_ENV_VAR
+    secrets:
+      - name: "MY_API_TOKEN"
+        valueFrom: "arn:aws:secretsmanager:us-east-1:123456789012:secret:FOO-AbCdEf:token::"
+      - name: "MY_PASSWORD"
+        valueFrom: "arn:aws:secretsmanager:us-east-1:123456789012:secret:FOO-AbCdEf:password::"
+    secrets_tags:
+      - "my_tag_name"
+```
+
+<ReferenceTable>
+  <ReferenceTableItem propertyName="container_context.ecs.env_vars">
+    A list of keys or key-value pairs to include in the task (if a value is not
+    specified, the value will be pulled from the agent task). In the example
+    above, `FOO_ENV_VAR` will be set to `foo_value` and the `BAR_ENV_VAR` will
+    be set to whatever value it has in the agent task.
+  </ReferenceTableItem>
+  <ReferenceTableItem propertyName="container_context.ecs.secrets">
+    Individual secrets specified using the [same structure as the ECS
+    API](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Secret.html).
+  </ReferenceTableItem>
+  <ReferenceTableItem propertyName="container_context.ecs.secrets_tags">
+    A list of tag names. Any secrets tagged with _any_ of those tag names in AWS
+    Secrets Manager will be included in the launched tasks. These secrets must
+    be key-value dictionaries.
+  </ReferenceTableItem>
+</ReferenceTable>
+
+If you're using the Dagster Cloud Github action, the `container_context` key can also be set for each location in your `dagster_cloud.yaml` file, using the same format.
+
+---
+
+### Per-job configuration: Resource limits
+
+You can use job tags to customize the CPU and Memory of every run for that job:
+
+```py
+from dagster import job, op
+
+@op()
+def my_op(context):
+  context.log.info('running')
+
+@job(
+  tags = {
+    "ecs/cpu": "256",
+    "ecs/memory": "512",
+  }
+)
+def my_job():
+  my_op()
+```
+
+[Fargate tasks only support certain combinations of CPU and Memory.](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html)
+
+## Per-deployment configuration
```

Comment:
this was moved from the manually provisioning page, also added some missing fields